tools/doc/node_modules/.bin/marked

```diff
@@ -0,0 +1 @@
+../marked/bin/marked
```

Comment:
Can you do this to remove node modules?

```
cd tools/doc && echo 'node_modules' > .gitignore && rm -rf node_modules && git add -A
```

Reply:
We have other `tools` directory `node_modules` subdirectories listed as ignored in the top-level `.gitignore` so putting it there is probably the way to go.

Reply:
Also, we'd need to add an `npm install` step for that directory in `Makefile` and `vcbuild.bat` similar to `lint-md-build`.

Reply:
Is it going to cause a problem to run `npm install` each time in the Makefile?
I'm still new to the build tools 😅

Reply:
@nodejs/build ^^^^^

Reply:
@styfle @Trott Probably not a big issue, but would obviously be better to avoid doing so (would cause longer build times)

Reply:
> Probably not a big issue, but would obviously be better to avoid doing so (would cause longer build times)

I guess that might explain checking in the node_modules directory. :-D